src/app/components/user-list-main/highlight.directive.ts

```diff
@@ -0,0 +1,18 @@
+import { Directive, ElementRef, Input } from '@angular/core';
+
+@Directive({
+    selector: '[appHighlight]',
+    standalone: true,
+})
+export class HighlightDirective {
+   
+    constructor(
+        private el: ElementRef<HTMLElement>,
+    ) { }
+
+    @Input()
+    set appHighlight(age: number) {
```

Comment:
Add another input parameter to define age limit:

`@Input({required: true}) ageLimit: number;`